user/language_features/configuration.md

```diff
@@ -56,3 +63,38 @@ gauge_clear_state_level = spec
 ```
 This clears the objects after the execution of each specification, so that new objects are created for next execution.
 
+
+## CSharp
+
+CSharp Specific configuration changes can be made in the ***env/default/default.properties*** file.
```

Comment:
I think these are reporting related properties which are applicable to all gauge projects irrespective of the language(CSharp/Java/Ruby).